### PR TITLE
added automapperTest and fixed profile

### PR DESCRIPTION
**Pull request by TheseusDeus**
closing #75

### CODE REVIEW

**Review by elisabethnetland — COMMENTED** [review]
I think this looks good! Would recommend someone with more knowledge in this area to approve :)

File: GiEnJul.Test/InfrastructureTests/AutoMapperTest.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (RonHaland) [inline GiEnJul.Test/InfrastructureTests/AutoMapperTest.cs:3]: These are not in use?

File: GiEnJul/Models/Person.cs [hunk 2]
```diff
@@ -16,8 +16,7 @@ public Person(string partitionKey)
         public int Age { get; set; }
         public Gender Gender { get; set; }
 
-        [JsonIgnore]
-        public Recipient Family { get; set; }
+
```
Comment (RonHaland) [inline GiEnJul/Models/Person.cs:19]: Remove unnecessary newlines?

**Review by RonHaland — DISMISSED** [review]
Approved with some small suggestions 👍

**Review by Sommerli — APPROVED** [review]

File: GiEnJul/Models/Person.cs (hunk 2 above)
Comment (TheseusDeus) [reply GiEnJul/Models/Person.cs:19]: true that can be removed

File: GiEnJul.Test/InfrastructureTests/AutoMapperTest.cs (hunk 1 above)
Comment (TheseusDeus) [reply GiEnJul.Test/InfrastructureTests/AutoMapperTest.cs:3]: yea, forgot to remove after generating class file

**Review by Sommerli — APPROVED** [review]

**Review by joffe97 — APPROVED** [review]

**Review by RonHaland — APPROVED** [review]

**Pull request merged**